lib/ui/lobby.dart

```diff
@@ -94,33 +96,13 @@ class _LobbyScreenState extends State<LobbyScreen> {
                           Expanded(
                             flex: 15,
                             child: SizedBox(
-                              height: appBarHeight * 0.75,
+                              height: appBarHeight * 0.8,
                               width: screenWidth * 0.55,
-                              child: TextField(
-                                controller: myController,
+                              child: AppTextField(
+                                textEditingController: myController,
+                                labelText: 'insertRoomCode'.tr,
+                                autofocus: false,
                                 keyboardType: TextInputType.number,
-                                inputFormatters: <TextInputFormatter>[
-                                  FilteringTextInputFormatter.digitsOnly
-                                ],
```

Comment:
And what are your thoughts on keeping the `inputFormatters`?

This prevents any non-digit character from being input into the text field.